k6/README.md

```diff
@@ -0,0 +1,21 @@
+# Hauki load testing
```

Comment:
Would it be easier to understand the purpose of the folder if the name was api-perf-tests etc. instead k6?